node/src/logging.rs

```diff
@@ -269,18 +271,24 @@ pub fn init_with_config(config: &LoggingConfig) -> anyhow::Result<()> {
         _ => write!(writer, "; {}={:?}", field, value),
     });
 
+    let filter = EnvFilter::new(
+        env::var(LOG_CONFIGURATION_ENVVAR)
+            .as_deref()
+            .unwrap_or("info"),
```

Comment:
what about all the logs coming from our dependencies? cc @Fraser999

Reply:
This is 100% the same as setting the `RUST_LOG` variable to `info`, which is our common practice. So while I am not sure what the behavior was before, I assume it is unchanged :)

Reply:
@goral09 I haven't seen any third party crates logging at `info` that I recall.  A couple have `debug` level, but at `info` we should be fine afaik.

Reply:
Thanks.